app/dashboard/courses/[courseId]/VideoList.tsx

```diff
@@ -0,0 +1,43 @@
+"use client";
+import { getCourseVideosById } from "@/app/api/courses/courses.api";
+import { convertSecondsToMinutes } from "@/app/configs/utils/format";
+import { useQuery } from "@tanstack/react-query";
+
+export const VideoList = ({ id }: { id: string }) => {
+  const { data: videos } = useQuery({
+    queryKey: ["videos", id],
+    queryFn: () => getCourseVideosById(id),
+    enabled: Boolean(id),
+  });
+
+  const sortedVideosByOrder = videos?.sort((a, b) => a.order - b.order);
+
+  return (
+    <div className="space-y-6">
+      {sortedVideosByOrder?.map(
```

Comment:
probably intead of using videos?.something... it will be better to handle errors and if you did not received videos render something like error components instead of nothing like now)